tests/MessageTest.php

```diff
@@ -0,0 +1,147 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Mailer\Tests;
+
+use DateTime;
+use DateTimeImmutable;
+use DateTimeInterface;
+use Yiisoft\Mailer\File;
+use Yiisoft\Mailer\Message;
+
+final class MessageTest extends \PHPUnit\Framework\TestCase
+{
+    public function testDefaultValues(): void
+    {
+        $message = new Message();
+        $this->assertSame('utf-8', $message->getCharset());
+        $this->assertSame([], $message->getFrom());
+        $this->assertSame([], $message->getTo());
+        $this->assertSame([], $message->getReplyTo());
+        $this->assertSame([], $message->getCc());
+        $this->assertSame([], $message->getBcc());
+        $this->assertSame('', $message->getSubject());
+        $this->assertNull($message->getDate());
+        $this->assertSame(3, $message->getPriority());
+        $this->assertSame('', $message->getReturnPath());
+        $this->assertSame('', $message->getSender());
+        $this->assertSame('', $message->getTextBody());
+        $this->assertSame('', $message->getHtmlBody());
+        $this->assertSame([], $message->getAttachments());
+        $this->assertSame([], $message->getEmbeddings());
+        $this->assertSame([], $message->getHeaders());
+        $this->assertNull($message->getError());
+    }
+
+    public function testCharset(): void
```

Comment:
```suggestion
    public function testWithCharset(): void
```

Reply:
Would be more clear IMO (the same applies to other similar tests).

Reply:
done